science/gr-iridium/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+categories          science comms
+platforms           darwin macosx
+license             GPL-3+
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+description         Iridium burst detector and demodulator.
+long_description    ${description}
+
+github.setup muccc gr-iridium eeaf8157dd78d5394bd7feefa3a1a5dd67af8037
+version   20190722-[string range ${github.version} 0 7]
+checksums rmd160 909c3ad0e0bdb12d53666d0ad85b6b95d8dcab80 \
+          sha256 e0fe908b32e209b7b2e1a6b3f92d0a12ad4de5eb6ed517f632afdc8c5a87a9e5 \
+          size   143041
+revision  0
+
+# use C++11
+compiler.cxx_standard 2011
+
+depends_build-append \
+    port:swig-python \
+    port:pkgconfig
+
+depends_lib-append \
+    port:boost \
+    path:lib/libgnuradio-runtime.dylib:gnuradio
+
+# specify the Python dependencies
+depends_lib-append port:python27
+# specify that Python version to use
+configure.args-append \
+    -DPYTHON_EXECUTABLE=${frameworks_dir}/Python.framework/Versions/2.7/bin/python2.7 \
+    -DPYTHON_INCLUDE_DIR=${frameworks_dir}/Python.framework/Versions/2.7/Headers \
+    -DPYTHON_LIBRARY=${frameworks_dir}/Python.framework/Versions/2.7/Python \
+    -DGR_PYTHON_DIR=${frameworks_dir}/Python.framework/Versions/2.7/lib/python2.7/site-packages
+
+configure.args-append \
+    -DDOXYGEN_DOT_EXECUTABLE= \
+    -DDOXYGEN_EXECUTABLE= \
+    -DCMAKE_MODULES_DIR=share/cmake
+
+variant docs description "Install ${name} documentation" {
+
+    depends_build-append \
+        port:doxygen \
+        path:bin/dot:graphviz
+
+    configure.args-delete \
+        -DDOXYGEN_DOT_EXECUTABLE= \
+        -DDOXYGEN_EXECUTABLE=
+
+    configure.args-append \
+        -DDOXYGEN_DOT_EXECUTABLE=${prefix}/bin/dot \
+        -DDOXYGEN_EXECUTABLE=${prefix}/bin/doxygen
```

Comment:
Use `configure.args-replace` instead of `-append` and `-delete`.

Reply:
I know about that, but I prefer to split the operation in two commands for readiness. This is obviously my point of view. Also, seeing the tcl code of `-replace`/`-append`/`delete` there is not performance penalty.